automations/js/src/last_week_tonight.mjs

```diff
@@ -69,16 +76,48 @@ const closedIssuesQ = (repo) =>
 const getItemsHtml = (title, items) => {
   if (!items.length) return []
 
+  // Get a unique list of stack labels
+  const stacks = [
+    ...new Set(
+      items
+        .map((item) =>
+          item.labels
+            .map((label) => label.name)
+            .filter((name) => name.startsWith('🧱 stack'))
+        )
+        .flat()
+    ),
+  ].sort()
+  console.log(stacks)
+
+  // Aggregate items by stack
+  let itemsByStack = {}
+
+  for (const stack of stacks) {
+    console.log(stack)
+    console.log(stack.split(':'))
+    const stackName = stack.split(':')[1].trim()
+    itemsByStack[stackName] = items
+      .filter((item) => item.labels.map((label) => label.name).includes(stack))
+      .sort((a, b) => a.number - b.number)
+  }
+
   return [
     `<h3>${title}</h3>`,
-    '<ul>',
-    ...items.map((item) => {
-      const href = item.html_url
-      const number = `#${item.number}`
-      const title = escapeHtml(item.title)
-      return `<li><a href="${href}">${number}</a>: ${title}`
-    }),
-    '</ul>',
+    // Produce a list of elements for each stack, then combine them all
+    ...Object.entries(itemsByStack)
+      .map(([stackName, items]) => [
+        `<h4>${stackNameMap[stackName] || stackName.replace(/\b[a-z]/g, (match) => match.toUpperCase())}</h4>`,
```

Comment:
Appreciate the obsessive attention towards display name.

Reply:
I didn't want them to look funky! haha